test/core/models/ocelot/stanzas/ValueStanzaSpec.scala

```diff
@@ -99,29 +102,47 @@ class ValueStanzaSpec extends BaseSpec {
     )
     .as[JsObject]
 
+  val validValueStanzaWithListJson: JsObject = Json
+    .parse(
+      s"""{
+         |  "type": "${stanzaType}",
+         |  "values": [
+         |    {
+         |      "type": "${listType}",
+         |      "label": "${listLabel}",
+         |      "value": "${listValue}"
+         |    }
+         |  ],
+         |  "next": ["${next}"],
+         |  "stack": ${stack}
+         |}
+    """.stripMargin
+    )
+    .as[JsObject]
+
   "ValueStanza" must {
 
-    "deserialise from json" in {
+    "deserialize from json" in {
 
       val stanza: ValueStanza = validValueStanzaJson.as[ValueStanza]
 
       stanza.stack shouldBe false
       stanza.next.length shouldBe 1
-      stanza.next(0) shouldBe next
+      stanza.next.head shouldBe next
       stanza.values.length shouldBe 2
-      stanza.values(0) shouldBe Value(Scalar, pageNameLabel, pageName)
-      stanza.values(1) shouldBe Value(Scalar, pageUrlLabel, pageUrl)
+      stanza.values.head shouldBe Value(ScalarType, pageNameLabel, pageName)
+      stanza.values(1) shouldBe Value(ScalarType, pageUrlLabel, pageUrl)
     }
 
-    "serialise to json" in {
-      val stanza: ValueStanza = ValueStanza(List(Value(Scalar, "LabelName", "/")), Seq("4"), true)
+    "serialize to json" in {
```

Comment:
lol, so we going for "s" now haha

Reply:
Yes, as defined by the Shorter Oxford English dictionary.